docs/quickstart-apt.md

```diff
@@ -0,0 +1,577 @@
+# Install with APT
+
+Use the Percona repositories to install using APT.
+
+## Prerequisits
+
+* Either use `sudo` or run as root
+
+* Stable Internet access
+
+## Installation steps
+
+The "expected results" depend on the operating system. The following examples are based on Ubuntu 22.04.
```

Comment:
```suggestion
The "expected results" depend on the operating system. The following examples are based on Ubuntu 22.04.
{.power-number}
```

Reply:
Since you've added {.power-number} in quickstart-docker.md, add them to all the lists

Reply:
"expected results" -> `expected results`